### PR TITLE
Final Capstone Project: Home page and About page

### PR DESCRIPTION
**Pull request by ndorrh**
This project is a part of microverse training program. It was built using HTML5, CSS, JavaScript, and BS5.
It is a web application for a web development boot camp. In this project I implemented the following:

1) **The home page** with all its markup and then use BS5 classes for styling the home page.
I also use some custom CSS to overwrite Some of the BS5 classes.
I also used JavaScript to dynamically generate the featured Instructors Section of the home page

2) **The about page:** with all its markup and used BS5 classes for styling and some custom CSS to overwrite some of the BS5 classes

3)**The close button** with its markup and styling and Added javaScript.

4) **Video description** link [Video](https://www.loom.com/share/777ee4f15993418ea31e0fac91c2907c)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -1,12 +1,211 @@
...
+          <a href=" class" class="about-hover"
+            ><img
+              src="images/about_2011_hover.png"
+              alt="about 2011"
+              class="w-100"
+          /></a>
+        </div>
+      </section>
+
+      <!--Footer-->
```
Comment (Munsa1) [inline about.html:185]: - [ ] Above the footer there is supposed to be a partner section for both the mobile and desktop view, kindly add it as it is part of the design.

Your design
image

Given design
image

- [ ] Also fix the color of the footer , it is white and not the color which is currently be used.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @ndorrh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 2]
```diff
@@ -90,20 +92,20 @@
...
             alt="Bootcamp banner"
           />
           <img
-            src="images/Blog-2-2.png"
-            class="w-75 d-none d-md-block"
+            src="images/about_title.png"
+            class="w-50 d-none d-md-block"
             alt="Bootcamp banner"
           />
           <span class="line"></span>
```
Comment (ichala) [inline about.html:111]: - [ ] great job removing the old image, but this title should be an HTML code styled with CSS and not an image, kindly try to remove the image and replace it with HTML Heading and try to style it like the template

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @ndorrh , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No linters errors✅ 
- Added partner section ✅ 


 ## Required Changes ♻️

- [ ] _**check the comments below**_


## Optional suggestions

- [x] N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -1,12 +1,322 @@
...
+          <h3 class="heading-small">"Hello, Sharing World"</h3>
+          <h1 class="heading-big">CREATIVE COMMONS <br>GLOBAL SUMMIT<br> 2015</h1>
+        </div>
+        <p class="intro-text">
+          Welcome to our 2022 web development Boot camp. The only course you
+          need to learn web development - HTML, CSS, JS, Node, and More!
+        </p>
+
+        <div class="event-date">
+          <h3 class="date">Starting on the (THU)15.10.2022 ~ 16(FRI)</h3>
```
Comment (OLIPLICHE) [inline index.html:109]: ## Optional:
- Please you need to increase the font size. Your page will be more attractive if you follow the design(size) provided by Cindy Shin(The author of the original design). I would like to emphasize that this project can land you in a new career as Frontend Dev. Kindly take this Optional Serious
- 
Capture

- Your Design:
Capture

**Review by OLIPLICHE — APPROVED** [review]
Hi @ndorrh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

##Highlights
- The required Changes were implemented (✅)) 👏
- The web is working fine👏👏


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -1,12 +1,324 @@
...
+          <img
+            src="images/Blog-2-2.png"
+            class="w-100 d-md-none"
+            alt="Bootcamp banner"
+          />
+          <img
+            src="images/Blog-2-2.png"
+            class="w-75 d-none d-md-block"
+            alt="Bootcamp banner"
+          />
```
[the remark this answers is not in the document]
Comment (ndorrh) [reply index.html]: Thanks for the review

**Pull request merged**